content/blog/2023/fastapi-path-operations-for-django-developers.md

```diff
@@ -24,10 +24,12 @@ on our recent experience converting an internal project from Django to FastAPI.
 [FastAPI]: https://fastapi.tiangolo.com/
 
 1. FastAPI Path Operations for Django Developers (this article)
-2. SQLAlchemy for Django Developers (coming soon)
+2. [SQLAlchemy for Django Developers]
 3. Testing a FastAPI Application (coming soon)
 4. How To Use FastAPI Dependency Injection Everywhere (coming soon)
 
+[SQLAlchemy for Django Developers]: /2023/10/20/sqlalchemy-for-django-developers/
```

Comment:
Is there a more generic way to get this links? Maybe linking to a post just by name and letting 11ty add the right date

Reply:
Not that I'm aware of...